FOREIGN-KEY-PROTECTION-STATUS.md

```
@@ -0,0 +1,342 @@
+
+| Entity | Unit Tests | Feature Tests | Total | Status |
+|--------|-----------|---------------|-------|--------|
+| Client | 8 | 8 | 16 | ✅ Complete |
+| Product | 7 | 8 | 15 | ✅ Complete |
+| Task | 9 | 11 | 20 | ✅ Complete |
+| Invoice | 0 | 11 | 11 | ✅ Complete |
+| TaxRate | 0 | 0 | 0 | ⏳ Pending |
+| Unit | 0 | 0 | 0 | ⏳ Pending |
+| Family | 0 | 0 | 0 | ⏳ Pending |
+| PaymentMethod | 0 | 0 | 0 | ⏳ Pending |
+| InvoiceGroup | 0 | 0 | 0 | ⏳ Pending |
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)